docs/source/usersguide/input.rst

```diff
@@ -1192,6 +1234,24 @@ Each ``material`` element can have the following attributes or sub-elements:
 
     *Default*: None
 
+  :macroscopic:
+    The ``macroscopic`` element is similar to the ``nuclide`` element, but,
+    recognizes that some multi-group libraries may be providing material
+    specific macroscopic cross sections instead of always providing nuclide
+    specific data like in the continuous-energy case.  To that end, the
+    macroscopic element has attributes/sub-elements called ``name``, and ``xs``.
+    The ``name`` attribute is the name of the cross-section for a
+    desired nuclide while the ``xs`` attribute is the cross-section
+    identifier. One example would be as follows:
+
+    .. code-block:: xml
+
+        <macroscopic name="UO2" xs="71c" />
```

Comment:
Hmm, xs was intended to mimic temperature dependencies just like it kind of is used for CE.  I haven't gotten far enough to have to care about temperature dependent library so I in a way left that capability there as a placeholder.